acme/api/account.go

```diff
@@ -125,9 +125,10 @@ func NewAccount(w http.ResponseWriter, r *http.Request) {
 		}
 
 		acc = &acme.Account{
-			Key:     jwk,
-			Contact: nar.Contact,
-			Status:  acme.StatusValid,
+			Key:            jwk,
+			Contact:        nar.Contact,
+			Status:         acme.StatusValid,
+			LocationPrefix: linker.GetLink(r.Context(), acme.AccountLinkType, ""),
```

Comment:
This line of code is used twice and it may not be immediately obvious what it's doing. Can you create an unexported function that generates the prefix, use that in both places, and add a comment what the format is?